include/board_config_manager.h

```diff
@@ -0,0 +1,32 @@
+#ifndef BOARD_CONFIG_MANAGER_H
+#define BOARD_CONFIG_MANAGER_H
+
+#include "configuration_enum.h"
+#include "config_manager.h"
+
+namespace rosflight_firmware
+{
+class BoardConfigManager
+{
+public:
+
+  // Get the largest number that is valid for the configuration of a given device
+  // This number is inclusive, i.e. a value of 2 means 0, 1, or 2 are valid
+  // For devices that are not configurable, returns 0
```

Comment:
Thanks for documenting! Could we convert these to doxygen-style comments though?